[MTM-65672] Release notes for cors for public api. (#4102)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2026-21-0-cors-support-for-public-api.md

---
date: ''

> title: CORS Support Added for Public API Endpoints

Reviewer: ```suggestion
title: CORS support added for public API endpoints
```

change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Platform services
component:
  - value: component-JlFdtOPva
    label: REST API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-65672
version: 2026.21.0
---

> Added the `Access-Control-Allow-Origin` header to responses from unauthenticated API endpoints to ensure proper Cross-Origin Request support, aligning them with existing authenticated APIs.

Reviewer 2: ```suggestion
Enabled CORS (Cross-Origin Resource Sharing) for unauthenticated REST API endpoints by adding the `Access-Control-Allow-Origin` HTTP response header. This ensures consistency with authenticated APIs and allows browser-based applications to access unauthenticated REST API endpoints like public platform metadata.
```

> Enabled CORS (Cross-Origin Resource Sharing) for unauthenticated REST API endpoints by adding the `Access-Control-Allow-Origin` HTTP response header. This ensures consistency with authenticated APIs and allows browser-based applications to access unauthenticated REST API endpoints like public platform metadata.

Reviewer: ```suggestion
CORS (Cross-Origin Resource Sharing) has been enabled for unauthenticated REST API endpoints by adding the `Access-Control-Allow-Origin` HTTP response header. This ensures consistency with authenticated APIs and allows browser-based applications to access unauthenticated REST API endpoints like public platform metadata.
```
